### ISSUE TITLE
External mutation doesn't work with VictoryZoomContainer

### ISSUE THREAD

**Issue by sarah-clements** [FormidableLabs/victory#1366]
# Bugs and Questions

### Checklist

- [x] This is not a `victory-native` specific issue. (Issues that only appear in `victory-native` should be opened [here](https://github.com/FormidableLabs/victory-native/issues/new))

- [x] I have read through the [FAQ](https://formidable.com/open-source/victory/docs/faq) and [Guides](https://formidable.com/open-source/victory/guides/) before asking a question

- [x] I am using the latest version of Victory

- [x] I've searched open issues to make sure I'm not opening a duplicate issue

### The Problem

When passing the VictoryZoomContainer to the containerComponent prop, external event mutations don't apply to the last mutated datapoint. If VictoryContainer is used or no container prop at all, this problem does not occur.

### Reproduction
Click on a datapoint to highlight it. Move the mouse to another datapoint and the highlighted style should be removed from all datapoints but it does not happen when the zoom container is used (I know one wouldn't normally trigger an external mutation with the mouseOver event, but it was done to simplify the demo). See the code sandbox [here](https://codesandbox.io/s/basic-victory-example-zm6xe?fontsize=14)

**Comment by jillian-cascade** [comment]
@sarah-clements did you figure out a solution to this issue? I am experiencing a very similar issue with this.

**Comment by sarah-clements** [comment]
> @sarah-clements did you figure out a solution to this issue? I am experiencing a very similar issue with this.

No, I didn't spend more time on it after filing the bug.

**Comment by jillian-cascade** [comment]
Fair enough. I did find the solution for this bug though, so I'll just post here since someone else may run into it.

Using the original example I've linked the solution code sandbox [here](https://codesandbox.io/s/external-mutation-bug-1no29).

Basically you just need to include the `externalEventMutations` and `events` properties on the parent chart component vs. defining them directly on the child components. This also means you must specify the `childName` that you want to event to fire on in both the `events` and in the `externalEventMutations`. More info here: https://formidable.com/open-source/victory/guides/events/#nested-component-events
